src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -830,16 +830,27 @@ void network::execute_primitive(const std::shared_ptr<primitive_inst>& primitive
     _events.insert({id, ev});
 }
 
+bool network::should_use_mutable_input(const program_node& node) {
```

Comment:
Reaaaaally sorry, but what about `is_mutable_input` ?

Reply:
how about decalre this as a lambda function if this is not to be used elswhere?

Reply:
Agree, I'll update it.